generate_testdata.sh

```diff
@@ -113,3 +114,6 @@ scaffold_test_project project-v2-addon 2
 scaffold_test_project project-v3 3-alpha go/v3-alpha
 scaffold_test_project project-v3-multigroup 3-alpha go/v3-alpha
 scaffold_test_project project-v3-addon 3-alpha go/v3-alpha
+scaffold_test_project project-v3-config 3-alpha go/v3-alpha "--component-config"
+scaffold_test_project project-v3-multigroup-config 3-alpha go/v3-alpha "--component-config"
+scaffold_test_project project-v3-addon-config 3-alpha go/v3-alpha "--component-config"
```

Comment:
Could we not only add more 1 scaffold such as : `project-v3-component-config`? I think it would be enough to ensure/check it. IMO add more 3 testdata sample will lead us to have 9 samples which can start to make hard to keep it maintained. WDYT?

Reply:
Agreed
```suggestion
scaffold_test_project project-v3-config 3-alpha go/v3-alpha --component-config
```

Reply:
Sounds good to me 👍